stacks/styleguide.prx.org.yml

```diff
@@ -0,0 +1,208 @@
+# stacks/styleguide.prx.org.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: styleguide.prx.org application running in Docker
+Mappings:
+  Shared:
+    Project:
+      name: styleguide.prx.org
+    Container:
+      name: styleguide-nginx
+Conditions:
+  CreateStagingResources: !Equals [!Ref EnvironmentType, Staging]
+  CreateProductionResources: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  # VPC ########################################################################
+  VPC:
+    Type: "AWS::EC2::VPC::Id"
+  # Load Balancer ##############################################################
+  PlatformALBDNSName:
+    Type: String
+  PlatformALBFullName:
+    Type: String
+  PlatformALBCanonicalHostedZoneID:
+    Type: String
+  PlatformALBHTTPListenerArn:
+    Type: String
+  PlatformALBHTTPSListenerArn:
+    Type: String
+  PlatformALBListenerPriorityPrefix:
+    Type: String
+  # ECS Cluster ################################################################
+  ECSCluster:
+    Type: String
+  ECSServiceAutoscaleIAMRoleArn:
+    Type: String
+  ECSServiceIAMRole:
+    Type: String
+  # Misc #######################################################################
+  OpsDebugMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  EcrRegion:
+    Type: String
+  SecretsBase:
+    Type: String
+  ContainerMemory:
+    Type: String
+  ContainerCpu:
+    Type: String
+  # Shared ENV #################################################################
+  # Styleguide ECS ################################################################
+  StyleguideEcrImageTag:
+    Type: String
+  # Styleguide ENV ################################################################
+  StyleguideSecretsVersion:
+    Type: String
+Resources:
+  StyleguideLogGroup:
+    Type: "AWS::Logs::LogGroup"
+    Properties:
+      RetentionInDays: 14
+  StyleguideALBTargetGroup:
```

Comment:
I'm not sure whether Styleguide needs any load balancing at all, but I've retained it from Metrics.